packages/manager/modules/pci/src/projects/new/payment/components/add/controller.js

```diff
@@ -33,11 +33,11 @@ export default class PciProjectNewPaymentMethodAddCtrl {
     this.ovhPaymentMethodHelper = ovhPaymentMethodHelper;
     this.OVH_PAYMENT_METHOD_TYPE = OVH_PAYMENT_METHOD_TYPE;
 
-    const { currency, ovhSubsidiary } = this.coreConfig.getUser();
+    const { currency, ovhSubsidiary, country } = this.coreConfig.getUser();
     this.registrationCharges = new Intl.NumberFormat(
       LANGUAGE_OVERRIDE[ovhSubsidiary]
         ? LANGUAGE_OVERRIDE[ovhSubsidiary]
-        : ovhSubsidiary.toLowerCase(),
+        : country,
```

Comment:
**question:** What is the need to change from `ovhSubsidiary` to `country`?

Reply:
I changed  ovhSubsidiary to country because when the user is asian the function  ovhSubsidiary.toLowerCase() return 'asia' ans when we pass 'asia' as a parameter to Intl.NumberFormat() we get an error but when we use a country code it works